appdirs.py

```diff
@@ -13,8 +13,8 @@
 # - Mac OS X: http://developer.apple.com/documentation/MacOSX/Conceptual/BPFileSystem/index.html
 # - XDG spec for Un*x: http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html
 
-__version_info__ = (1, 4, 3)
-__version__ = '.'.join(map(str, __version_info__))
+__version__ = "1.4.3"
+__version_info__ = tuple(int(segment) for segment in __version__.split("."))
```

Comment:
This will fail if the version number has non-integers in it, like 'dev' or 'alpha' markers. I don't know if this is ever the case for this package. This may be an option then:

```
__version__ = "1.4.3.dev0"  # as example

def intify(value):
    try:
        return int(value)
    except ValueError:
        return value

__version_info__ = map(intify, __version__.split('.'))
```